tools/FatesPFTIndexSwapper.py

```diff
@@ -10,12 +10,13 @@
 # =======================================================================================
 
 import numpy as np
+from numpy import *
 import sys
 import getopt
 import code  # For development: code.interact(local=locals())
 from datetime import datetime
 from scipy.io import netcdf
-import matplotlib.pyplot as plt
+#import matplotlib.pyplot as plt
```

Comment:
I just ran into this same issue -- that tools/FatesPFTIndexSwapper.py imports pyplot but doesn't actually use it anywhere, so its an unneeded dependency that complicates things.  suggest delete this line entirely.

Reply:
yup, agreed